Rego/EXOConfig.rego

```diff
@@ -242,16 +242,25 @@ tests[{
     "ReportDetails" : ReportDetailsString(Status, ErrorMessage),
     "RequirementMet" : Status
 }] {
-    ErrorMessage := "Wildcard domain (\"*\") in shared domains list, enabling sharing with all domains by default"
-
-    Status := count(SharingPolicyAllowedSharing) == 0
+    ContactsSharingPolicies := SharingPolicyContactsAllowedAllDomains
```

Comment:
Shouldn't `ContactsSharingPolicies` be the ActualValue in line 241 instead of `input.sharing_policy`?